MatrixSDK/MXSession.h

```diff
@@ -499,6 +500,11 @@ FOUNDATION_EXPORT NSString *const kMXSessionNoRoomTag;
  */
 @property (nonatomic, readonly) MXEventStreamService *eventStreamService;
 
+/**
+ The module that manages location sharing.
+ */
+@property (nonatomic, readonly) MXLocationService *locationService;
```

Comment:
We had a non-optional `threadingService` at https://github.com/matrix-org/matrix-ios-sdk/blob/3857e9d59c2c0da071b8702794b8cb9f1a732a7f/MatrixSDK/Contrib/Swift/MXSession.swift#L16-L18
You may want to have something similar. Totally optional.

Reply:
Currently it's seen as `var locationService: [MXLocationService]! { get }` in Swift. It would be great to have nullability defined in `MXSession` but many work to do for sure. But yes it can be a good way to clarify nullability.